examples/chart/teleport-cluster/templates/proxy/_config.common.tpl

```diff
@@ -70,4 +70,7 @@ proxy_service:
     uri: {{ .Values.acmeURI }}
   {{- end }}
 {{- end }}
+{{- if and .Values.ingress.enabled (semverCompare ">=14.0.0" (include "teleport-cluster.version" .)) }}
```

Comment:
this check will break on alpha releases, you need to set `semverCompare ">= 14.0.0-0" ...` (see [the docs of this terrible behaviour](http://masterminds.github.io/sprig/semver.html))

I'm also not sure it's really useful to gate features in the chart. We can leverage the chart versionning system to avoid having to do this each time we add a new feature. However, we'll need to be careful when backporting the ingress support not to pick this as well.

Reply:
Yeah - this was pretty much a deliberately defensive decision on my part. If this PR is backported verbatim, it won't inadvertently break anything because of this check. It's going to need to be changed to 13.2.6 or whatever version we're at when the other PR is merged anyway, so we might as well have the glue in place?

Reply:
I'm also concerned that people will still `helm repo update` after the v14 release and then `helm install teleport/teleport-cluster --set teleportVersionOverride=13.3.0`, which will end up breaking their setups without a version gate. Obviously we _tell_ everyone to do `helm install --version 13.3.0 teleport/teleport-cluster` instead, but maybe it's better to be careful.